client/my-sites/current-site/index.jsx

```diff
@@ -113,7 +113,6 @@ const CurrentSite = React.createClass( {
 	},
 
 	previewSite: function( event ) {
-		analytics.ga.recordEvent( 'Sidebar', 'Clicked View Site' );
```

Comment:
I'd probably keep that in as it's the same kind of action. 

I'm however a bit surprised it's tracking just GA. Are the other items in the sidebar not tracked in Tracks + GA?